bot.js

```diff
@@ -34,18 +34,50 @@ controller.hears(['play'], 'direct_message,direct_mention,mention', function(bot
     setTimeout(function(){
         questions = generateQuestion();
         console.log(questions);
+        console.log(message.channel)
+        controller.storage.channels.save({id: message.channel, questions:questions}, function(err){});
         bot.reply(message, "Solve: "+ questions[0]+ " " + questions[1]+ " " + questions[2]+ " " + questions[3])
         }, 4000);
 });
 
+controller.hears(['hint'], 'direct_message,direct_mention,mention', function(bot, message) {
+    data = controller.storage.channels.get(message.channel, function(err, data){
+        if(data && data.questions) {
+            var problem_input = data.questions.join(" ")
+            var solution = solver.solve(problem_input, 31, ['+','-','*','/', '**']);
```

Comment:
@YosuaMichael using your solver 😄 . Will be good if it can accept array of numbers.